roles/istio/templates/default-seldon-gateway.yaml.j2

```diff
@@ -23,6 +23,10 @@ spec:
       protocol: HTTPS
     tls:
       mode: SIMPLE
+      {% if istio_self_signed_certs | bool %}
       serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
       privateKey: /etc/istio/ingressgateway-certs/tls.key
+      {% else %}
+      credentialName: {{ istio_ingress_cert_secret }}
```

Comment:
Only comment would be that this can be an empty string, which is clearly a configuration error.
It'd probably be easier to remove `istio_self_signed_certs` and just have `istio_ingress_cert_secret` with docs/a comment saying "leave this empty if you want self-signed certs".

Reply:
Yeah I might rethink this a bit anyway since I'll also need to make changes to certmanager. Will update this PR later when I've figured out the best way to do it.

Reply:
Worth changing it to a draft until you're ready then?